packages/components/src/components/dialog/dialog.custom.styles.ts

```diff
@@ -0,0 +1,54 @@
+import { css } from 'lit';
+
+export default css`
+  :host {
+    /**
+     * Used as the default padding for the drawer title
+     */
+    --header-spacing: var(--syn-spacing-large) var(--syn-spacing-x-small) var(--syn-spacing-large) var(--syn-spacing-large);
+
+    /**
+     * Defines the default body spacing
+     */
+    --body-spacing: var(--syn-spacing-medium) var(--syn-spacing-large);
+
+    /**
+     * Defines the default footer spacing
+     */
+    --footer-spacing: var(--syn-spacing-medium) var(--syn-spacing-large) var(--syn-spacing-large) var(--syn-spacing-medium);
+  }
+
+  .dialog__panel {
+    border: var(--syn-spacing-4x-small) solid var(--syn-panel-border-color);
+    border-radius: var(--syn-border-radius-none);
+    box-shadow: var(--syn-shadow-large);
+  }
+
+  .dialog__header-actions {
+    align-items: flex-start;
+    gap: var(--syn-spacing-x-small);
+    padding: var(--syn-spacing-large) var(--syn-spacing-small) var(--syn-spacing-large) 0;
+  }
+
+  /**
+   * We need this to make sure we are big enough as defined in the layout
+   * When omitting this statement, we will get rounding problems via line-height
+   */
+  .dialog__title,
+  .header-actions {
```

Comment:
Same as with the drawer. We don't have any `header-actions` class in the dialog component. Therefore the min-height is never added somewhere. Should be `dialog__header-actions` instead

```suggestion
  .dialog__header-actions {
```